vllm/v1/kv_cache_interface.py

```diff
@@ -112,15 +119,30 @@ def max_memory_usage_bytes(self, vllm_config: VllmConfig) -> int:
 
 
 @dataclass
-class KVCacheTensor:
+class KVCacheTensorBase:
     """
     A dataclass for specifying how the workers should initialize the KV cache
-    for a layer. Only contains the size of KV cache for that layer for now. Will
-    be extended to support multiple layers sharing the same memory pool.
+    for a layer.
+    """
+    pass
+
+
+@dataclass
+class KVCacheNewTensor(KVCacheTensorBase):
```

Comment:
Some of the tests might need to update to this naming, KVCacheNewTensor, e.g. test_kv_cache_utils.py

Reply:
Thanks for pointing it out. This PR is just a POC, so I didn't fix the tests.